src/components/Modal/Modal.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Modal.scss';
+const Modal = ({ handleClose, show, children }) => {
+    const showHideClassName = show ? "modal modal-display-block" : "modal modal-display-none";
+    return (
+      <div className={showHideClassName}>
+        <section className="modal-main">
+          {children}
+          <button onClick={handleClose}>close</button>
+        </section>
+      </div>
```

Comment:
We are not given many many options to user here. If user click close button gets an empty page which is kind of intimidating. I guess we need something like `Enable Geeolocation` button to trigger permission API like here https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API/Using_the_Permissions_API

Reply:
Permission API is automatically triggered when location is tried to be retrieved. In any case user should be able to use the app. Permissions can be asked once and afterwards users can decide whether their positions will be showed on the map.